cache-http.sh

```diff
@@ -15,10 +15,18 @@ if [ -z $INPUT_LOCK_FILE ]; then
     exit;
 fi
 
+echo "check connection"
+curl \
+    -u $INPUT_BASIC_AUTH_USERNAME:$INPUT_BASIC_AUTH_PASSWORD \
+    -X GET \
+    -x "$INPUT_HTTP_PROXY" \
+    $INPUT_CACHE_HTTP_API/health \
+
 shaLockfile=`openssl sha1 $INPUT_LOCK_FILE |awk '{print $2}'`
 shaInstallCommand=`echo $INPUT_INSTALL_COMMAND|openssl sha1|awk '{print $2}'`
+shaDestinationFolder=`echo $INPUT_DESTINATION_FOLDER|openssl sha1|awk '{print $2}'`
 
-tarFile=$RUNNER_OS-$INPUT_VERSION-$shaInstallCommand-$shaLockfile.tar.gz
+tarFile=$RUNNER_OS-$INPUT_VERSION-$shaInstallCommand-$shaLockfile-$shaDestinationFolder.tar.gz
```

Comment:
I don't think destination folder has any effect on the dependency caching. It is the folder where the tar will be untarred to.

Reply:
In my example, in addition to the node_modules folder, I have a second one in which I also have artifacts that I want to cache. Both folders will be tar and endtared. However, I did add the second folder until the second Workflow run, so there was already a tar in the cache that contained only one folder. Is certainly just an edgecase. I could have thrown away the cache, too.

Reply:
Thanks for the clarification. I have pushed the tag v2.1 https://github.com/kevincobain2000/action-cache-http/releases/tag/v2.1